pom.xml

```diff
@@ -36,15 +36,22 @@
     </developers>
 
     <properties>
-        <aws-java-sdk.version>1.11.817</aws-java-sdk.version>
+        <aws-java-sdk.version>2.13.64</aws-java-sdk.version>
+        <commons.logging.version>1.1.3</commons.logging.version>
```

Comment:
I believe we dont need this, Please correct me if I am wrong.

Reply:
You are right.

            <groupId>software.amazon.awssdk</groupId>
            <artifactId>utils</artifactId>

includes slf4j.